ports/windows/mpconfigport.h

```diff
@@ -229,12 +229,9 @@ typedef long mp_off_t;
 #if MICROPY_ENABLE_SCHEDULER
 // Use 1mSec sleep to make sure there is effectively a wait period:
 // something like usleep(500) truncates and ends up calling Sleep(0).
-#define MICROPY_EVENT_POLL_HOOK \
-    do { \
-        extern void mp_handle_pending(bool); \
-        mp_handle_pending(true); \
-        msec_sleep(1.0); \
-    } while (0);
+#define MICROPY_INTERNAL_WFE(TIMEOUT_MS) msec_sleep(MIN(1.0, (double)(TIMEOUT_MS)))
```

Comment:
What happens when TIMEOUT_MS is -1?

Reply:
Derp this should be MAX so it always sleeps at least 1ms, good catch.